### DISCUSSION TITLE
Selenium-recaptcha-solver

### DISCUSSION THREAD

**Discussion by osadchiidev** [community/community#63033] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I can't run it.

**Comment by nightfury-toothless** [comment]
I can help you.

**Comment by nightfury-toothless** [comment]
Please contact me via my gmail: venusdev16@gmail.com

**Comment by osadchiidev** [comment]
Please let me know your skill

**Comment by nightfury-toothless** [comment]
I am very familiar with Selenium-recaptcha-solver

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
